projects/packages/search/src/customizer/class-customizer.php

```diff
@@ -32,8 +26,6 @@ public function __construct() {
 	/**
 	 * Initialize Customizer controls.
 	 *
-	 * @since 8.3.0
-	 *
 	 * @param WP_Customize_Manager $wp_customize Customizer instance.
 	 */
 	public function customize_register( $wp_customize ) {
```

Comment:
I suspect the two `require_once` are not needed anymore 😄

Reply:
Good catch!